test/expected/size_utils.out

```diff
@@ -46,29 +46,72 @@ INSERT INTO "two_Partitions"("timeCustom", device_id, series_0, series_1) VALUES
-        3 | "_timescaledb_internal"."_hyper_1_3_chunk" | {timeCustom,device_id} | {"[1257985728000000000,1257988320000000000)","[1073741823,2147483647)"} |        8192 |       98304 |        8192 |      114688 | 8192 bytes | 96 kB      | 8192 bytes | 112 kB
-        4 | "_timescaledb_internal"."_hyper_1_4_chunk" | {timeCustom,device_id} | {"[1257892416000000000,1257895008000000000)","[0,1073741823)"}          |        8192 |       98304 |        8192 |      114688 | 8192 bytes | 96 kB      | 8192 bytes | 112 kB
+ chunk_id |                chunk_table                 |       dimensions       | dimension_types |                                 ranges                                  | table_bytes | index_bytes | toast_bytes | total_bytes 
+----------+--------------------------------------------+------------------------+-----------------+-------------------------------------------------------------------------+-------------+-------------+-------------+-------------
+        1 | "_timescaledb_internal"."_hyper_1_1_chunk" | {timeCustom,device_id} | {bigint,text}   | {"[1257892416000000000,1257895008000000000)","[1073741823,2147483647)"} |        8192 |      114688 |        8192 |      131072
+        2 | "_timescaledb_internal"."_hyper_1_2_chunk" | {timeCustom,device_id} | {bigint,text}   | {"[1257897600000000000,1257900192000000000)","[1073741823,2147483647)"} |        8192 |      106496 |        8192 |      122880
+        3 | "_timescaledb_internal"."_hyper_1_3_chunk" | {timeCustom,device_id} | {bigint,text}   | {"[1257985728000000000,1257988320000000000)","[1073741823,2147483647)"} |        8192 |       98304 |        8192 |      114688
+        4 | "_timescaledb_internal"."_hyper_1_4_chunk" | {timeCustom,device_id} | {bigint,text}   | {"[1257892416000000000,1257895008000000000)","[0,1073741823)"}          |        8192 |       98304 |        8192 |      114688
+(4 rows)
+
+SELECT * FROM chunk_relation_size_pretty('"public"."two_Partitions"');
+ chunk_id |                chunk_table                 |                             dimensions                              | dimension_types |                                   ranges                                    | table_size | index_size | toast_size | total_size 
+----------+--------------------------------------------+---------------------------------------------------------------------+-----------------+-----------------------------------------------------------------------------+------------+------------+------------+------------
+        1 | "_timescaledb_internal"."_hyper_1_1_chunk" | {timeCustom,_timescaledb_internal.get_partition_for_key(device_id)} | {bigint,text}   | {"'1257892416000000000' - '1257895008000000000'","1073741823 - 2147483647"} | 8192 bytes | 112 kB     | 8192 bytes | 128 kB
```

Comment:
I changed the dimension output from the initial PR to show that it is a function of the column like `_timescaledb_internal.get_partition_for_key(device_id)`.

Reply:
I'm mixed on that change re: dimension output.  Seems to just add wordiness without additional clarity?

Might be better to show something like (although not sure if possible):  `{ timeCustom::Interval, device_id::Hash }`

Reply:
I added the comment as I am not sure either. The thing is that the hash function can be changed by the user and this code would show the name of the actual hash function used. Although that might be a rare special case so I am happy to change to your suggestion.